central/declarativeconfig/singleton.go

```diff
@@ -0,0 +1,16 @@
+package declarativeconfig
+
+import "github.com/stackrox/rox/pkg/sync"
+
+var (
+	once     sync.Once
+	instance Manager
+)
+
+// Singleton provides the instance of Manager to use.
+func Singleton() Manager {
```

Comment:
Let's rename it to `ManagerSingleton`, otherwise we get usage like `declarativeconfig.Singleton()` which is quite confusing